runner/src/examples.rs

```diff
@@ -655,7 +655,7 @@ pub fn build_wasm_module(name: &str, target: &Target, example_name: &str) -> Ste
                         // `--out-dir` is unstable and requires `-Zunstable-options`.
                         "-Zunstable-options".to_string(),
                         "build".to_string(),
-                        "--target=wasm32-unknown-unknown".to_string(),
+                        "-Ztimings --target=wasm32-unknown-unknown".to_string(),
```

Comment:
Apparently this ends up being squashed with the --target flag; it needs to be defined separately as its own string, it's not enough to have a space in this string.

Reply:
Greetings of the day @tiziano88,

I've tested the -Z timings locally [here](https://github.com/shikharvashistha/blockchained/blob/807c6078f408e8fc3ab54fdd33e0b2ffdb4b44ac/.github/workflows/rust.yml#L50) && [here](https://github.com/shikharvashistha/blockchained/runs/3573330564?check_suite_focus=true) and came to a conclusion that it is currently only available on nightly channel as stated [here](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#timings) too. So in my opinion if this project uses nightly channel for CI then only it would be beneficial to introduce the -Z timings flag. What do you suggest ?

Thanks & Regards
@shikharvashistha